entire_vds_pipeline.py

```diff
@@ -446,9 +450,13 @@ def export_to_elasticsearch(
     samples_in_vds = set(vds.sample_ids)
     matched = sample_query.intersection(samples_in_vds)
     if len(matched) < len(sample_query) and args.ignore_extra_sample_ids_in_tables:
-        logger.warning('Found only {0} out of {1} samples specified for ID remapping'.format(len(matched), len(sample_query)))
+        logger.warning('Found only {0} out of {1} samples specified for subsetting'.format(len(matched), len(sample_query)))
+        logger.info('Failed to find the following requested IDs for subsetting: {}'.format(
+            [x for x in sample_query.difference(samples_in_vds)]))
     elif len(matched) < len(sample_query):
-        raise ValueError('Found only {0} out of {1} samples specified for ID remapping'.format(len(matched), len(sample_query)))
+        raise ValueError('Found only {0} out of {1} samples specified for subsetting'.format(len(matched), len(sample_query)))
+        logger.info('Failed to find the following requested IDs for subsetting: {}'.format(
```

Comment:
not sure if github is just displaying indents incorrectly, but 
logger.info appears to be blocked by raise ValueError